docs/reference/esql/task-management.asciidoc

```diff
@@ -9,7 +9,7 @@ You can list running {esql} queries with the <<tasks,task management API>>:
 
 [source,console,id=esql-task-management-get-all]
 ----
-GET /_tasks?pretty&detailed&group_by=parents&human&actions=*data/read/esql
+GET /_tasks?pretty=true&human=true&detailed=true&group_by=parents&actions=*data/read/esql
```

Comment:
This change isn't necessary, the original works just fine.

Reply:
From description, the original does not work for users on ESS 🙂 . And as ESS bug has been open 5y I would prefer to add in this slight change since the API works both ways for ES. 🙏

Reply:
I would rather we fixed this bug in the ESS API console rather than in the docs here. At the very least, it'd be better to document this limitation in the API console. If we did decide to add the unnecessary `=true` in our docs, we would want to (a) do it everywhere, not just the few places this PR fixes, and (b) include some validation to make sure we have caught every instance and don't add any more instances in future.